web/modules/custom/demo_content/demo_content.module

```diff
@@ -13,15 +13,15 @@ use Drupal\og\OgMembershipInterface;
  * Implements hook_ENTITY_TYPE_create().
  */
 function demo_content_rdf_entity_create(EntityInterface $entity) {
-  if ($entity->bundle() !== 'collection') {
+  if (!in_array($entity->bundle(), ['collection', 'solution'])) {
     return;
   }
-
-  // Set 'facilitator' user as a facilitator for the collection.
-  $facilitator = user_load_by_name('facilitator');
+  $user_name = $entity->bundle() . '_facilitator';
+  $facilitator = user_load_by_name($user_name);
```

Comment:
Haha, sneaky!!

On a side note, we're losing valuable documentation here. The docblock does not contain any explanation why this hook implementation is necessary, and now the only line that explains it is removed.